src/instruments/src/EFB/TODCalculator/Data/Data.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * A32NX
+ * Copyright (C) 2020-2021 FlyByWire Simulations and its contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { connect } from 'react-redux';
+import Input from "../../Components/Form/Input/Input";
+import React from "react";
+import Card from "../../Components/Card/Card";
+import Divider from "../../Components/Divider/Divider";
+import {TOD_CALCULATOR_REDUCER} from "../../Store";
+import {setTodData} from "../../Store/action-creator/tod-calculator";
+import {TOD_CALCULATION_TYPE} from "../../Enum/TODCalculationType.enum";
+
+const Data = ({
+    currentAltitude,
+    targetAltitude,
+    calculation: { type: calculationType, input: calculationInput },
+    setTodData,
+    ...props
+}) => {
+    const calculationTypes = [
+        {label: 'Distance', rightLabel: 'NM', type: TOD_CALCULATION_TYPE.DISTANCE},
+        {label: 'Vertical speed', rightLabel: 'FT/m', type: TOD_CALCULATION_TYPE.VERTICAL_SPEED}
```

Comment:
![image](https://user-images.githubusercontent.com/929769/107510063-bd051400-6ba3-11eb-911c-cf91c9d0c10b.png)

I would suggest following the ICAO definition and use exactly `ft/min` (in lowercase).

The uppercase `NM` seems correct:

![image](https://user-images.githubusercontent.com/929769/107510190-e0c85a00-6ba3-11eb-83ef-243cf4005c83.png)



Source: https://www.theairlinepilots.com/forumarchive/quickref/icao/annex5.pdf

Reply:
good point! changing